x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/multi_signal_inventory/table/no_entities_empty_state.tsx

```diff
@@ -27,15 +27,22 @@ import {
   AssociateServiceLogs,
   CollectServiceLogs,
 } from '../../../../shared/add_data_buttons/buttons';
+import { useKibana } from '../../../../../context/kibana_context/use_kibana';
+import { ApmPluginStartDeps, ApmServices } from '../../../../../plugin';
 
 export function NoEntitiesEmptyState() {
   const { core } = useApmPluginContext();
+  const { services } = useKibana<ApmPluginStartDeps & ApmServices>();
   const { basePath } = core.http;
   const [userHasDismissedCallout, setUserHasDismissedCallout] = useLocalStorage(
     'apm.uiNewExperienceCallout',
     false
   );
 
+  useEffect(() => {
```

Comment:
Nit: You can use `useEffectOnce` from 'react-use/lib/useEffectOnce' for similar events like [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/infra/public/components/asset_details/template/flyout.tsx/#L36) for example